controllers/checlusterrestore/backup_data_restorer.go

```diff
@@ -123,6 +123,9 @@ func cleanPreviousInstallation(rctx *RestoreContext, dataDir string) (bool, erro
 		rctx.cheCR = cheCR
 	}
 
+	// Drop databases, if any, to handle case of different Postgres version (otherwise clean Che will fail to start due to ald data in volume)
```

Comment:
```suggestion
	// Drop databases, if any, to handle case of different Postgres version (otherwise clean Che will fail to start due to old data in volume)
```